service/http.balancer/pom.xml

```diff
@@ -98,6 +98,24 @@
             <groupId>org.kaazing</groupId>
             <artifactId>k3po.junit</artifactId>
         </dependency>
+        <dependency>
```

Comment:
You can see this being done in the top level pom: https://github.com/kaazing/gateway/blob/develop/pom.xml.  I included an excerpt at the bottom.  And perhaps you don't need to specify them in this pom at all because of that (you will have to confirm that independently).  

> what does that do ?

The main dependency section of the pom is pulled in while compiling the code and test.  If you put the scope as test, it is only pulled in while compiling the tests.  

The scripts in the specification project are actually only needed at run time, and specifically only for the runtime of the k3po driver which is launched during the plugin.  There isn't really a way to extend the scripts, and there is generally no code in them that you need while compiling the tests.

The dependency section of the plugin is only pulled in when running the plugin.  With k3po, the junit part of the test tells the driver (which is launched by the plugin) to find a script.  The driver needs to search its class path.

So it's just a minor fix saying, eclipse and test compiling does not need that jar and dependencies

```
  <plugin>
                <dependencies>
                    <dependency>
                        <groupId>org.kaazing</groupId>
                        <artifactId>specification.http</artifactId>
                        <version>${k3po.version}</version>
                    </dependency>
                    <dependency>
                        <groupId>org.kaazing</groupId>
                        <artifactId>specification.httpx</artifactId>
                        <version>${k3po.version}</version>
                    </dependency>
                    <dependency>
                        <groupId>org.kaazing</groupId>
                        <artifactId>specification.ws</artifactId>
                        <version>${k3po.version}</version>
                    </dependency>
                    <dependency>
                        <groupId>org.kaazing</groupId>
                        <artifactId>specification.tcp</artifactId>
                        <version>${k3po.version}</version>
                    </dependency>
                    <dependency>
                        <groupId>org.kaazing</groupId>
                        <artifactId>specification.wse</artifactId>
                        <version>${k3po.version}</version>
                    </dependency>
                    <dependency>
                        <groupId>org.kaazing</groupId>
                        <artifactId>specification.x-kaazing-ping-pong</artifactId>
                        <version>${k3po.version}</version>
                    </dependency>
                </dependencies>
                <groupId>org.kaazing</groupId>
                <artifactId>k3po-maven-plugin</artifactId>
                <version>${k3po.version}</version>
                <configuration>
                    <skipTests>${skipITs}</skipTests>
                </configuration>
                <executions>
                    <execution>
                        <goals>
                            <goal>start</goal>
                            <goal>stop</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
```